examples/nlp/sentence_classification_with_bert.py

```diff
@@ -63,6 +63,13 @@
 hidden_size = pretrained_bert_model.local_parameters["hidden_size"]
 tokenizer = BertTokenizer.from_pretrained(args.pretrained_bert_model)
 
+"""A processor function can be passed to SentenceClassificationDataDesc
+for customized data preparation. An example processor can be found at
+the bottom of this script.
+
```

Comment:
not a to-do for this PR but just a thought that we should probably allow the processor argument for every DataDesc class.